src/resources/UsageAnalytics/Read/Dimensions/DimensionsInterfaces.ts

```diff
@@ -1,13 +1,21 @@
 import {DimensionStatus, DimensionType} from '../../../Enums';
 
 export interface DimensionModel {
+    // A collection of possible names to use when querying the statistics service with this dimension
```

Comment:
Thanks for the docs!

```suggestion
    /**
     * A collection of possible names to use when querying the statistics service with this dimension
     */
```

In the single line format, the comment isn't perceived as documentation by most IDEs (tested with vscode here)
- `/**/`
![image](https://user-images.githubusercontent.com/35579930/116927730-0ce6fa80-ac2a-11eb-8e22-e163d952f7d2.png)
- `//`
![image](https://user-images.githubusercontent.com/35579930/116927780-1cfeda00-ac2a-11eb-956c-71efb5524344.png)

Reply:
Got it, I'll change them to multiline 👍